cassovary-core/src/test/scala/com/twitter/cassovary/graph/bipartite/BipartiteGraphSpec.scala

```diff
@@ -155,95 +154,99 @@ class BipartiteGraphSpec extends Specification {
     graph = new BipartiteGraph(leftSide, rightSide, BipartiteGraphDir.Both)
   }
 
-  "Bipartite Graph Single Side Input" definedAs bipartiteExampleSingleSide should{
+  "Bipartite Graph Single Side Input" should {
     "have loaded correct number of nodes and edges" in {
```

Comment:
you can use:
val graph = bipartiteExampleSingleSide() before the 'in' statements and not have to change any of the 'in' tests